js/Makefile

```diff
@@ -389,32 +355,14 @@ packages/go-bridge/android/libs/gobridge.aar: $(bridge_src)
 		./go/framework/bertybridge
 	touch $@
 
-packages/berty-app/android/app/debug.keystore:
-	cp packages/berty-app/node_modules/react-native/template/android/app/debug.keystore $@
-
-packages/storybook-mobile/android/app/debug.keystore:
-	cp packages/berty-app/node_modules/react-native/template/android/app/debug.keystore $@
-
-packages/berty-app/node_modules/react-native/react.gradle.orig: packages/berty-app/patch/react.gradle.patch
-	@echo "patching file $(patsubst %.orig, %, $@)"
-	@patch -f $(patsubst %.orig, %, $@) $< 1>/dev/null || true
```

Comment:
I don't know but it looks like a hack

Reply:
I think it was made to disable the debug related features on the different release targets we have.
The patch replace something like: `if target == release` by `if target != debug` because we have 3 differents release targets.
It's probably not the best way to do it but now with this change, all the debug features will be enabled on the different release targets.

Reply:
I don't know, maybe. I didn't do the initial setup. I'll see what to do once it's merge, after thinking about it, it won't hurt to start from scratch. ;)